Src/BootCamp.Chapter/Solution.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+
+public class Solution
+{
+    public int MyAtoi(string str)
+    {
+        int result = 0;
+        int sign = 1;
+
+        if (!string.IsNullOrEmpty(str) && str.Length > 0)
+        {
+            char[] chars = str.ToCharArray();
```

Comment:
And be simplified:
step 1: remove all whitespaces from string. str.Replace(" ", "")